controller/logs.go

```diff
@@ -68,18 +68,26 @@ func (c *Controller) LogsForDeployment(ctx context.Context, req *entity.Deployme
 				return err
 			}
 			// Current Logs fetched from server
-			currLogs := strings.Split(deploy.DeployLogs, "\n")
+			currLogs := strings.Split(fetchCurrentLogs(deploy), "\n")
 			// Diff logs using the line numbers as references
-			logDiff := currLogs[len(prevLogs)-1 : len(currLogs)-1]
+			idx := int(math.Max(float64(len(idxMp[deploy.Status])-1), 0.0))
```

Comment:
:neckbeard:

Reply:
You know, I was happier not knowing that was an emoji.